### ISSUE TITLE
Cannot Render KiCAD 3D Shapes

### ISSUE THREAD

**Issue by mobilinkd** [bartmcleod/VrmlParser#6]
@bartmcleod  I am having trouble rendering most of the KiCAD 3D parts available here:

https://github.com/KiCad/kicad-packages3D

```
   vrml.min.js:69 Uncaught TypeError: Cannot read property 'node' of undefined
       at v (vrml.min.js:69)
       at VrmlParser.Renderer.ThreeJs.render (vrml.min.js:74)
       at Object.onLoad (TNC3.html:225)
       at XMLHttpRequest.<anonymous> (three.min.js:674)
```

This is the first one I fail with: https://github.com/KiCad/kicad-packages3D/blob/master/Capacitor_SMD.3dshapes/C_0402_1005Metric.wrl

The parser renders the KiCAD generated PCB files just fine.  But once I create a merged board with even one part, the board no longer renders.

**Comment by bartmcleod** [comment]
I tested it your example and the cause of the issue is quite simple: near the beginning of the wrl file, two shapes are defined without a geometry. Their sole purpose is to define two materials, that can be re-used later in the file. As a work around, you could either specify a geometry, or move the declaration of the appearance to the first shape where they are actually used with a geometry. If the VRML 97 specification allows to have a shape with only an appearance and no geometry, then I will be happy to support that. Will look into it. For now, the workaround is to make sure that a Shape also has a geometry.

[Edit] At least a value of NULL for the geometry should be supported, which means absence of a geometry should also be supported.

**Comment by bartmcleod** [comment]
Fixed in 0.7.6
